_posts/2020-05-04-strangler-pattern.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: Strangler Pattern
+date: 2020-05-04
+tags: architecture patterns
+author: Tim Myerscough
+image: img/strangler-pattern/Strangler_fig_boulder_katandra.jpg
+---
+
+<center><a href="https://commons.wikimedia.org/wiki/File:Strangler_fig_boulder_katandra.jpg"> <img alt="strangler fig" src="/img/strangler-pattern/Strangler_fig_boulder_katandra.jpg" /></a></center>
+<br/>
+
+Technology is constantly evolving, so thinking in terms of the end goal is unrealistic.  Many companies struggle to keep up with the advances in technology.  If companies lag behind it increases commercial pressure from nimble new entrants, or competitors that can move faster.
+
+Evolutionary architecture, doesn't aim for an end state.  One tool for evolutionary architecture is the [Strangler Pattern](https://martinfowler.com/bliki/StranglerFigApplication.html).
+
+# The Context
+When considering migration to the cloud, [Infrastructure Modernisation](https://www.mechanicalrock.io/our-expertise/infrastructure-modernisation), or [Serverless technology](https://www.mechanicalrock.io/our-expertise/cloud-native-solutions) how can we move faster whilst dealing with legacy systems?
```

Comment:
"Whether considering a migration to the cloud, infrastructure modernisation, or serverless technology; how can we move faster whilst dealing with legacy systems?"

I feel it needs a semi-colon (or something) to separate the clauses.